tests/Offer/CommandHandlers/AddLabelHandlerTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\Testing\CommandHandlerScenarioTestCase;
+use Broadway\EventHandling\EventBus;
+use Broadway\EventStore\EventStore;
+use CultuurNet\UDB3\Calendar;
+use CultuurNet\UDB3\CalendarType;
+use CultuurNet\UDB3\Event\EventRepository;
+use CultuurNet\UDB3\Event\Events\EventCreated;
+use CultuurNet\UDB3\Event\Events\LabelAdded;
+use CultuurNet\UDB3\Event\EventType;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Label;
+use CultuurNet\UDB3\Label\LabelServiceInterface;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\Entity;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Label\ValueObjects\LabelName;
+use CultuurNet\UDB3\Label\ValueObjects\Privacy;
+use CultuurNet\UDB3\Label\ValueObjects\Visibility;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Offer\Commands\AddLabel;
+use CultuurNet\UDB3\Offer\OfferRepository;
+use CultuurNet\UDB3\Place\PlaceRepository;
+use CultuurNet\UDB3\Title;
+use PHPUnit\Framework\MockObject\MockObject;
+use ValueObjects\Identity\UUID;
+use ValueObjects\StringLiteral\StringLiteral;
+
+final class AddLabelHandlerTest extends CommandHandlerScenarioTestCase
+{
+    /**
+     * @var ReadRepositoryInterface|MockObject
+     */
+    private $labelRepository;
```

Comment:
I think this can be replaced with a local variable?

Reply:
Right! Fixed in 396f9d5